<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stackShow</class>
 <widget class="QDialog" name="stackShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lneExp">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_exp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>83</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stack</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_res">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>146</y>
     <width>31</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneStack">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneRes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
